src/constants.js

```diff
@@ -46,6 +46,11 @@ export const USER_ACTIVITY_REPORT_ENTITY_RELATIVE_DISTRIBUTION = "RelativeDistri
 export const USER_ACTIVITY_REPORT_ENTITY_SERVICE = "Service";
 export const USER_ACTIVITY_REPORT_ENTITY_USER = "InteractiveUser";
 export const USER_ACTIVITY_REPORT_ENTITY_USER_DISTRICT = "UserDistrict";
+
+export const STORAGE_KEY_SECONDARY_CALENDAR = "isSecondaryCalendarEnabled";
+export const DEFAULT_SECOND_CALENDAR_FORMAT = "DD-MM-YYYY";
```

Comment:
Consider creating an object DEFAULT with those values: 
```
const DEFAULT = {
SECOND_CALENDAR_FORMAT: "DD-MM-YYYY",
SECOND_CALENDAR_LANG: "en",
};
```
Just a suggestion, no need to apply if you find it useless.